Libraries/Components/DatePicker/DatePickerIOS.ios.js

```diff
@@ -123,7 +123,7 @@ var DatePickerIOS = React.createClass({
         <RCTDatePickerIOS
           ref={ picker => this._picker = picker }
           style={styles.datePickerIOS}
-          date={props.date.getTime()}
+          date={props.date}
```

Comment:
According to @nicklockwood this is actually wrong, React just verifies the wrong prop types somehow.